common/changes/office-ui-fabric-react/active-element-fix_2019-02-22-03-02.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Find active element within an iframe",
+      "type": "patch"
+    }
+  ],
+  "packageName": "office-ui-fabric-react",
+  "email": ""
```

Comment:
@aftab-hassan this should have been populated with your email of some sort. You can always use the `users.noreply.github.com` address if you wish. It populates our changelogs.